lib/backend/k8s/conversion.go

```diff
@@ -109,7 +109,7 @@ func (c converter) namespaceToProfile(ns *k8sapi.Namespace) (*model.KVPair, erro
 		if k == policyAnnotation {
 			np := namespacePolicy{}
 			if err := json.Unmarshal([]byte(v), &np); err != nil {
-				return nil, err
+				return nil, goerrors.New(fmt.Sprint("Failed to parse annotation: %s", err))
```

Comment:
nit: I think it should be lowercase F in the Failed....   (I think that's usually the case for error text since it gets chained)